app/src/main/res/values/strings.xml

```diff
@@ -181,10 +181,13 @@ be temporary, and you can try again later.</li>
     <string name="turbo_mode_disabled_toast">Turbo Mode disabled</string>
     <string name="onboarding_turbo_mode_title">Turbo Mode helps you browse faster</string>
     <string name="onboarding_turbo_mode_body">Turbo Mode automatically blocks trackers and ads to help get you where you\'re going faster. If there\'s a speed bump – like a site that isn\'t behaving the way you expect – just turn Turbo Mode off.</string>
+    <string name="onboarding_turbo_mode_body2">Turbo Mode automatically blocks trackers and ads to help get you where you’re going faster. If there’s a speed bump – like a site that isn’t behaving the way you expect – just turn Turbo Mode off.</string>
```

Comment:
Did you try building this? I think you need to escape the apostrophes.

Reply:
Yeah, I was surprised as well, but it looks like the curved apostrophes don't need to be escaped.

Reply:
To be clear, I did build to double check.

Reply:
@Baron-Severin I feel like delphine has previously said we shouldn't have curly apostrophes in strings, but I'm checking right now.

Reply:
Sounds like it was the opposite: everything should be curly. lgtm.